karate-tests/src/test/java/patients/healthcareWorker/updatePatientDetails/replace.feature

```diff
@@ -0,0 +1,66 @@
+@sandbox
+Feature: Patch patient - Replace data
+
+  Background:
+    * def utils = call read('classpath:helpers/utils.feature')
+    * def accessToken = karate.callSingle('classpath:patients/healthcareWorker/auth-redirect.feature').accessToken
+    * url baseURL
+
+  Scenario: Replace attribute of an object
+    # To replace the attribute of an object, you need to provide the id of the object you want to replace
+    # in a preceding operation. (This is simlar to, but different from removing an object - we call 
+    # "replace" instead of "test")
+    * def nhsNumber = karate.env == 'mock' ? '9000000009' : '5900056597'
```

Comment:
Can the process of setting the NHS number and retrieving it be called in the Background portion like you've done elsewhere?